docker-compose.yaml

```diff
@@ -9,3 +9,8 @@ services:
       - "3000:3000"
     environment:
       NODE_ENV: production
+      CONTACT_DETAILS_QUEUE_ADDRESS: ${CONTACT_DETAILS_QUEUE_ADDRESS:-notset}
```

Comment:
Ok for the moment, we need to look at what we're going to do locally instead of using the Azure Service Bus. Redis streams maybe.

Reply:
Yes indeed.

Reply:
Seems like 2 ASB emulators came along at once: [https://defra-digital-team.slack.com/archives/C07UN8QRZ9N/p1732043932229359?thread_ts=1731922094.144029&cid=C07UN8QRZ9N](url)

Reply:
> Seems like 2 ASB emulators came along at once: [https://defra-digital-team.slack.com/archives/C07UN8QRZ9N/p1732043932229359?thread_ts=1731922094.144029&cid=C07UN8QRZ9N](url)

That link is weird, the link itself is pointing at `https://github.com/DEFRA/ffc-grants-eligibility-checker/pull/url` for me.